[2017-10-22 23:41:40,445] bcr.Node_<Node_:a0001>:ERROR: Unrecoverable error in node process: FileNotFoundError(2, 'No such file or directory')
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/da/sim.py", line 977, in _delayed_start
    return self.run()
  File "src/bcr.py", line 723, in run
    config_dict = read_config.parse_config_file(infile)
  File "src/read_config.py", line 7, in parse_config_file
    with open(filename, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'config/replica-dict-objects.txt'
